modules/user-access/pages/enabling-user-authentication.adoc

```diff
@@ -56,7 +56,7 @@ To enable RESTPP authentication, set the `RESTPP.Factory.EnableAuth` parameter t
 
 . As the TigerGraph Linux user, run the following command:
 +
-.Enabling REST{pp} OAuth Authentication
+.Enabling REST{pp} SAML2.0 Authentication
```

Comment:
this is not correct, please just simply remove `OAuth` or `SAML2.0`.
SAML2.0 is for SSO in GSQL side, here is restpp Authentication which is not related to SSO.